docs/reactivity/reactive-components.md

```diff
@@ -0,0 +1,99 @@
+# Build reactive components with Oryx
+
+## Integrate with domain logic
+
+Components can use domain logic to fetch data. Domain services provide a convenient API to request data, e.g. in case of loading product data, the Product Service can be leveraged:
+
+```ts
+export class ProductPriceComponent extends LitElement {
+  protected productService = resolve(ProductService);
+  protected product$ = this.productService.get({ sku: '123' });
+}
+```
+
+The `ProductPriceComponent` should ideally not be aware of the SKU, as this is provided by the context; the SKU might be resolved from the URL or from the context, i.e. from a cart entry component. Oryx provides a context controller that can be used to resolve the SKU. Different domains have their own controller to resolve the right qualifier from the context. In case of the product domain we use the `ProductController`:
```

Comment:
How is SKU being provided by the context ideal as opposed to the component being aware of it? What's wrong with the component being aware?

Reply:
I've improved the docs to better articulate why components should not be aware.